src/Symfony/Component/Console/Command/Command.php

```diff
@@ -427,17 +433,18 @@ public function getNativeDefinition(): InputDefinition
     /**
      * Adds an argument.
      *
-     * @param int|null $mode    The argument mode: InputArgument::REQUIRED or InputArgument::OPTIONAL
-     * @param mixed    $default The default value (for InputArgument::OPTIONAL mode only)
+     * @param $mode    The argument mode: InputArgument::REQUIRED or InputArgument::OPTIONAL
+     * @param $default The default value (for InputArgument::OPTIONAL mode only)
+     * @param array|Closure(CompletionInput):array $suggestedValues The values used for input completion
      *
      * @throws InvalidArgumentException When argument mode is not valid
      *
      * @return $this
      */
-    public function addArgument(string $name, int $mode = null, string $description = '', mixed $default = null): static
+    public function addArgument(string $name, int $mode = null, string $description = '', mixed $default = null, array|\Closure $suggestedValues = null): static
```

Comment:
we need to access those via func_get_arg for BC
the default should be `[]`

Reply:
Done.

Reply:
The deprecation listener works well. How to hide this messages?
```
  1x: The "Symfony\Component\Console\Command\LazyCommand::addArgument()" method will require a new "array|\Closure $suggestedValues" argument in the next major version of its parent class "Symfony\Component\Console\Command\Command", not defining it is deprecated.
    1x in AboutCommandTest::testAboutWithReadableFiles from Symfony\Bundle\FrameworkBundle\Tests\Command\AboutCommand

  1x: The "Symfony\Component\Console\Command\LazyCommand::addOption()" method will require a new "array|\Closure $suggestedValues" argument in the next major version of its parent class "Symfony\Component\Console\Command\Command", not defining it is deprecated.
    1x in AboutCommandTest::testAboutWithReadableFiles from Symfony\Bundle\FrameworkBundle\Tests\Command\AboutCommand
```

Reply:
Genius. I added the `@param` annotation and the deprecation is gone.